app/pods/agenda/agendaitems/agendaitem/index/template.hbs

```diff
@@ -5,15 +5,6 @@
         {{t "agendaitem-case"}}
       </h3>
     </Auk::Toolbar::Item>
-    {{#if (await this.subcase.confidential)}}
-      <Auk::Toolbar::Item>
-        <Auk::Pill
-          data-test-route-agenda---agendaitem-index-confidential
```

Comment:
This test selector got removed, but is used in tests.
add this in agendaitem-titles template

Reply:
nieuwe selector gemaakt waar nodig